dali/operators/decoder/image_decoder.cc

```diff
@@ -144,6 +144,11 @@ the GPU implementation.
 According to the libjpeg-turbo documentation, decompression performance is improved by up to 14%
 with little reduction in quality.)code",
       false)
+  .AddOptionalArg("use_jpegturbo_upsampling",
```

Comment:
how about "jpeg_fancy_upsampling" ? 
Also, can you add this to the other implementation of nvjpeg?

Reply:
Done